Documentation/sources/enum-public-classes/enum-messaging-edge-event-type.md

```diff
@@ -15,7 +15,7 @@ public enum MessagingEdgeEventType {
     PUSH_APPLICATION_OPENED(4),
     PUSH_CUSTOM_ACTION(5);
 
-    final int value;
+final int value;
```

Comment:
nit: the tab seemed to get removed here. can it be re-added back to preserve the formatting?